cmd/manager/main.go

```diff
@@ -33,6 +33,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
 )
 
+var (
+	cloudConfig        = flag.String("cloud_config", "", "path to the GCE config")
+	machineSetupConfig = flag.String("machine_setup_config", "/etc/machinesetup/machine_setup_configs.yaml", "path to the machine setup config")
```

Comment:
+1 to flags!

I did a consistency check against flag naming in k/k; we seem to use `-` as the separator e.g. `cloud-config`.  (And we do use the name `cloud-config` for the equivalent file in k-c-m, so that's good)

Nit picking, but it's a pain to change later (although I think some of the flag libraries can normalize `_` => `-`?)

Reply:
Sure thing! Changed to "-"